From 156fdb5681927a8a39d00c53f97caeffa33f8f83 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 17 Jun 2020 14:04:52 +0200
Subject: [PATCH] Update AppStream metadata

- Specify launchable desktop file and developer name.
- Make metadata license more permissive to allow inclusion into distributions' AppStream database.
---
 resources/linux/birdfont.appdata.xml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/resources/linux/birdfont.appdata.xml b/resources/linux/birdfont.appdata.xml
index 35828092..e36853f9 100644
--- a/resources/linux/birdfont.appdata.xml
+++ b/resources/linux/birdfont.appdata.xml
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component>
 	<id>org.birdfont.birdfont</id>
-	<metadata_license>GPL-3.0+</metadata_license>
+	<launchable type="desktop-id">birdfont.desktop</launchable>
+	<metadata_license>CC0-1.0</metadata_license>
 	<project_license>GPL-3.0+</project_license>
 	<name>birdfont</name>
 	<summary>Font editor</summary>
@@ -29,4 +30,5 @@
 		<binary>birdfont</binary>
 	</provides>
 	<update_contact>johan.mattsson.m@gmail.com</update_contact>
+	<developer_name>Johan Mattsson</developer_name>
 </component>
